tgui/packages/tgui/interfaces/PlaneMasterDebug.tsx

```diff
@@ -931,10 +929,7 @@ const AddModal = (props) => {
     'currentPlane',
     {} as Plane,
   );
-  const [currentTarget, setCurrentTarget] = useLocalState<Plane>(
-    'currentTarget',
-    {} as Plane,
-  );
+  const [currentTarget, setCurrentTarget] = useState<Plane>({} as Plane);
```

Comment:
If the above on L386 is fine then I'd suggest casting that way instead of using `as`
```suggestion
  const [currentTarget, setCurrentTarget] = useState<Plane>({});
```

Reply:
Reverted with https://github.com/tgstation/tgstation/pull/80459/commits/796956a956bda82faa16413b9cd16361d43008a9